### PR TITLE
[IMP] l10n_fi,l10n_eu_oss: new 25.5% tax instead of 24 required

### PR DESCRIPTION
**Pull request by Megaaaaaa**
Description of the issues this commit addresses:

As from the first of september 2024, Finland will raise 24% taxes to 25.5%, those rates are currently not available in Odoo.

---

Desired behavior after this commit is merged:

New 25.5% taxes are available in Finland
OSS tax mapping is done for other countries to Finland The mapping from Finland to other countries is deliberately left for later as we don't want the mapping to 25.5% to tkae place before the 09/01/24.

---

task-4010731

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/l10n_fi/data/account_fiscal_position_template_data.xml [hunk 1]
```diff
@@ -225,22 +275,42 @@
...
+        <field name="position_id" ref="non_eu"/>
+        <field name="tax_src_id" ref="tax_dom_purchase_goods_25_5"/>
+        <field name="tax_dest_id" ref="tax_non_eu_purchase_goods_25_5"/>
+    </record>
     <record id="non_eu_purchase_goods_24" model="account.fiscal.position.tax.template">
         <field name="position_id" ref="non_eu"/>
         <field name="tax_src_id" ref="tax_dom_purchase_goods_24"/>
         <field name="tax_dest_id" ref="tax_non_eu_purchase_goods_24"/>
     </record>
+    <record id="non_eu_purchase_goods_14" model="account.fiscal.position.tax.template">
```
Comment (aliyatastemirova) [inline addons/l10n_fi/data/account_fiscal_position_template_data.xml:288]: you have some duplicate ids here and below :slightly_smiling_face:

File: addons/l10n_fi/data/account_tax_report_line.xml [hunk 2]
```diff
@@ -10,6 +10,14 @@
         <field name="sequence">0</field>
         <field name="report_id" ref="vat_report"/>
     </record>
+    <record id="tax_report_sales_25_5" model="account.tax.report.line">
+        <field name="name">25.5 %:n vero</field>
+        <field name="code">sale_25_5</field>
+        <field name="tag_name">fi_301</field>
```
Comment (aliyatastemirova) [inline addons/l10n_fi/data/account_tax_report_line.xml:16]: shouldn't the tag name be different from the report line below? :thinking:

File: addons/l10n_fi/data/account_tax_report_line.xml [hunk 3]
```diff
@@ -10,6 +10,14 @@
...
+        <field name="sequence">-1</field>
```
Comment (aliyatastemirova) [inline addons/l10n_fi/data/account_tax_report_line.xml:17]: pretty sure it's better to resequence everything, but maybe this time we could do it in increments of 10: 10, 20, 30

**Review by aliyatastemirova — CHANGES_REQUESTED** [review]
thanks, lgtm, just a few comments
also, should we make 24% taxes inactive? :thinking: or not yet, because it's not September yet?

**Comment by Megaaaaaa** [comment]
Indeed 24% taxes shouldn't be made unavailable until september. I have another task which will modify the oss mapping and it should not be done before september so i will check with the PO if we want to include that in that task too.

File: addons/l10n_fi/data/account_tax_report_line.xml (hunk 2 above)
Comment (Megaaaaaa) [reply addons/l10n_fi/data/account_tax_report_line.xml]: Indeed, i couldn't find its use here but it's used in later versions so i will pay attention to that during FW

**Comment by aliyatastemirova** [comment]
robodoo r+

**Pull request closed without merging**